### PR TITLE
Upgrade Spotbugs plugin from 2.0.1 to latest 4.0.8

### PR DESCRIPTION
**Pull request by vorburger**
FINERACT-947

### CODE REVIEW

**Comment by vorburger** [comment]
@awasum given our past collaboration on initial FINERACT-702, would you like to review this?

**Review by awasum — APPROVED** [review]
Great work @vorburger . LGTM. Merging next after Post merge Travis task completes on another PR merge. (Merge this if I forget, please)

**Pull request merged**
